client/src/offline-settings/db.ts

```diff
@@ -0,0 +1,145 @@
+// WARNING - This file is duplicated at two locations:
+// - client/pwa/src/db.ts
+// - client/src/offline-settings/db.ts
+// Until we find a solution, keep both files in sync.
```

Comment:
I would feel better if we either copy this file in the build process or have a test that compares the hash of the two `db.ts` files.

Reply:
Added test in f0c1de7a0.